manifests/templates/file-host.yaml.in

```diff
@@ -36,6 +36,18 @@ spec:
         volumeMounts:
         - name: "images"
           mountPath: "/tmp/shared/"
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 80
+          initialDelaySeconds: 20
+          periodSeconds: 20
+        livenessProbe:
```

Comment:
I see a reason for a readiness probe. What's the motivation for the liveness probe? If the server dies (and we're using a deployment to run it) is there still a benefit to the liveness probe?

Reply:
Liveness is detects whether the internal process is running as expecting, is not dead locked or in some other state that would not cause the pod to fail, but cause disruption of service.